apis/sql/v1beta1/sqlinstance_types.go

```diff
@@ -441,7 +437,37 @@ type InstanceValueFrom struct {
 	SecretKeyRef *v1alpha1.SecretKeyRef `json:"secretKeyRef,omitempty"`
 }
 
+type BinLogCoordinates struct {
+	/* Name of the binary log file for a Cloud SQL instance. */
+	BinLogFileName string `json:"binLogFileName,omitempty"`
+
+	/* Position (offset) within the binary log file. */
+	BinLogPosition int64 `json:"binLogPosition,omitempty,string"`
+}
+
+type CloneSource struct {
+	/* Binary log coordinates, if specified, identify the position up to which the source instance is
+	cloned. If not specified, the source instance is cloned up to the most recent binary log coordinates. */
+	// +optional
+	BinLogCoordinates *BinLogCoordinates `json:"binLogCoordinates,omitempty"`
+
+	/* (SQL Server only) Clone only the specified databases from the source instance. Clone all databases if empty. */
+	// +optional
+	SQLDatabaseRefs []refsv1beta1.SQLDatabaseRef `json:"sqlDatabaseRefs,omitempty"`
```

Comment:
This is a tricky one.... should this be a ref or just a list of strings.  They have to be scoped to the parent source instance, right?  Refs does allow for dynamic naming, but not sure how likely that is.

Reply:
Yeah, good point. Fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2429